list.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'enumerable'
+# class with a enumerable methods
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    yield(@list)
+  end
```

Comment:
- [ ] Kindly fix the implementation of this method. The problem isn't in the testing written below the class, the problem is in the implementation of the `each` method.

- [HINT] The implementation of the `each` method should yield one item at a time. You can do this if you implement your `each` method to look something like the following:
```ruby
def each
   counter = 0
   while counter < @list.length
      yield(@list[counter])
      counter += 1
   end
end
```

- In this implementation, I iterated over the list with a `counter` and yielded each item in the list using `@list[counter]` at a time.